usr.sbin/kldxref/ef.h

```diff
@@ -37,6 +41,18 @@
 #define EF_LOOKUP_SYMBOL(ef, name, sym) \
     (ef)->ef_ops->lookup_symbol((ef)->ef_ef, name, sym)
 
+#if __has_feature(capabilities)
+#if defined(__riscv)
+#define	HAS_CAPRELOCS 1
+#define	DT_CAPRELOCS DT_RISCV_CHERI___CAPRELOCS
+#define	DT_CAPRELOCSSZ DT_RISCV_CHERI___CAPRELOCSSZ
+#elif defined(__mips__)
+#define	HAS_CAPRELOCS 1
+#define	DT_CAPRELOCS DT_MIPS_CHERI___CAPRELOCS
+#define	DT_CAPRELOCSSZ DT_MIPS_CHERI___CAPRELOCSSZ
+#endif
+#endif
```

Comment:
These already exist in the form of DT_CHERI___CAPRELOCS(SZ); sys/kern/link_elf.c gates on whether DT_CHERI___CAPRELOCS is defined